packages/frontend/src/components/ChatsList.js

```diff
@@ -1,27 +1,41 @@
 import { useState } from 'react';
 import { ListBox } from 'primereact/listbox';
 import { Card } from 'primereact/card';
+import { VirtualScroller } from 'primereact/virtualscroller';
 
-export default function ChatsList() {
+export default function ChatsList(props) {
 	// maybe make the parent handle the state.
-	const [selectedChat, setSelectedChat] = useState('')
+
+
+	// TODO: Make ListBox scrollable.
 	let friends = [
 		'PaxFilipinas',
 		'Senor Monikas',
-		'Toyota Prius International Fan Club'
+		'Toyota Prius International Fan Club',
+		'test',
+		'test1',
+		'test2',
+		'test3',
+		'test4',
+		'test5',
+		'test6',
+		// 'test7',
+		// 'test8',
+		// 'test9',
+		// 'test10'
 	]
 
 	// should have values with friends,
 	// group chats, etc. and proper icons
 	return (
-		<div className='flex flex-column w-3'>
-			<Card title='Your Friends' className='mb-2'></Card>
+		<div className='flex flex-column w-3 p-4'>
+			<Card title='Your Friends' className='surface-50 mb-3 shadow-3 border-noround'></Card>
 			<ListBox
 				filter
-				value={selectedChat}
-				onChange={(e) => setSelectedChat(e.value)}
+				onChange={(e) => props.onChatSelect(e.value)}
+				onClick={console.log('list box clicked')}
```

Comment:
This is not doing what you think it's doing!
Could you encapsulate this, like this:
`onClick={() => console.log('list box clicked')}`

Reply:
So we turn that into a function, instead of a statement!